x-pack/test/alerting_api_integration/security_and_spaces/group1/tests/alerting/backfill/find.ts

```diff
@@ -48,42 +48,49 @@ export default function findBackfillTests({ getService }: FtrProviderContext) {
       expect(data.duration).to.eql('12h');
       expect(data.enabled).to.eql(true);
       expect(data.start).to.eql('2023-10-19T12:00:00.000Z');
-      expect(data.end).to.eql('2023-10-20T12:00:00.000Z');
+      expect(data.end).to.eql('2023-10-25T12:00:00.000Z');
       expect(data.status).to.eql('pending');
       expect(data.space_id).to.eql(spaceId);
       expect(typeof data.created_at).to.be('string');
       testExpectedRule(data, ruleId, false);
-      expect(data.schedule).to.eql([
-        {
-          run_at: '2023-10-20T00:00:00.000Z',
-          status: 'pending',
-          interval: '12h',
-        },
-        {
-          run_at: '2023-10-20T12:00:00.000Z',
-          status: 'pending',
-          interval: '12h',
-        },
-      ]);
+      expect(data.schedule.length).to.eql(12);
+      data.schedule.forEach((sched: any) => {
```

Comment:
changing so status doesn't have to match `pending` (in case any backfill jobs complete)